hapi-fees/src/main/java/com/hedera/services/usage/crypto/CryptoOpsUsage.java

```diff
@@ -72,7 +72,7 @@ public void cryptoTransferUsage(
 		accumulator.addBpt(incBpt);
 
 		long incRb = numXfers * LONG_ACCOUNT_AMOUNT_BYTES;
-		incRb += TOKEN_ENTITY_SIZES.bytesUsedToRecordTokenTransfers(weightedTokensInvolved, weightedTokenXfers);
+		incRb += TOKEN_ENTITY_SIZES.bytesUsedToRecordTokenTransfers(weightedTokensInvolved, weightedTokenXfers, 0);
```

Comment:
Shouldn't we include `nft` transfer usages and not hardcode `0`?

Reply:
This cryptoTransferUsage is for legacy receipts. Also data differentiating fungibles from nfts is missing in CryptoTransferMeta. That's why i've include zero for nfts here